MaterialDesignThemes.Wpf/PopupBox.cs

```diff
@@ -579,12 +579,6 @@ private void AnimateChildrenIn(bool reverse)
                 uiElement.SetCurrentValue(RenderTransformOriginProperty, new Point(.5, .5));
                 uiElement.RenderTransform = transformGroup;
 
-                var opacityAnimation = new DoubleAnimationUsingKeyFrames();                
-                opacityAnimation.KeyFrames.Add(new EasingDoubleKeyFrame(0, absoluteZeroKeyTime, sineEase));
```

Comment:
This is removing the subtle fade in effect.  I think we need something like this here:
```
if (uiElement.IsEnabled)
{
```